language_server/imports.md

```diff
@@ -0,0 +1,342 @@
+## Import completions and intelligent registries
```

Comment:
Maybe we should use 1 `#` like the other files?

Reply:
Actually, most other sub pages use `##` instead of `#`.  It is only the top level "chapters" that use `#`.